src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -326,5 +336,13 @@ export const WalletOptions: Definitions<WalletConfig> = {
     default: () => ["m", "44'", "60'", "0'", "0"],
     legacyName: "hd_path",
     cliType: "string"
-  }
+  },
+  unlockAll: {
+    normalize,
+    cliDescription:
+      "Unlock all accounts irrespective of known or unknown status.",
```

Comment:
I can't think of better/clearer alternatives, but I wonder if this help text will make sense to our users.

I'm curious on other reviewers' thoughts on this

Reply:
I agree - the term "known or unknown status" seems to be an internal concern. We could probably get more information in there, but at the very least, just "Unlock all accounts" would be clearer.